### PR TITLE
sec: upcoming security announcement

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

**Comment by mhdawson** [comment]
@nodejs/website please review, would like to publish within the next hour (Although I'm in a meeting 3-4 so it may be 4 before I get back to it, so feel free to land if it looks ok)

**Review by MylesBorins — APPROVED** [review]
LGTM

**Comment by MylesBorins** [comment]
Should we also update the main header?

**Review by chris--young — APPROVED** [review]

File: locale/en/blog/vulnerability/december-2017-security-releases.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+date: 2017-12-04T19:30:00.617Z
+category: vulnerability
+title: Data Confidentiality/Integrity Vulnerability, September 2017
```
Comment (maclover7) [inline locale/en/blog/vulnerability/december-2017-security-releases.md]: December 2017

**Review by maclover7 — COMMENTED** [review]
one quick nit

**Comment by mhdawson** [comment]
@maclover7 thanks, good catch. Updated.

**Comment by mhdawson** [comment]
@MylesBorins we should probably define our policy on that.  If we should do it as part of the announce and release I'll update the instructions to reflect that.

File: locale/en/blog/vulnerability/december-2017-security-releases.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+layout: blog-post.hbs
+author: Michael Dawson
+---
+
+# Summary
+The Node.js project will be releasing new versions of 4.x, 6.x, 8.x and 9.x as soon as possible after the OpenSSL release, on or soon after December 8th UTC, to incorporate a security fix.
+
+# Data Confidentiality/Integrity Vulnerability
+
+All versions of 4.x, 6.x, 8.x and 9.x are vulnerable to an issue to be fixed in the forthcoming OpenSSL-1.0.2n released on Dec 7 see https://mta.openssl.org/pipermail/openssl-announce/2017-December/000108.html for more details. The severity of this vulnerability of Node.js is HIGH (due to the way Node.js uses the OpenSSL APIs) and users of the affected versions should plan to upgrade when a fix is made available.
```
Comment (lpinca) [inline locale/en/blog/vulnerability/december-2017-security-releases.md]: Nit: add a comma after Dec 7?

**Review by lpinca — APPROVED** [review]

**Comment by lpinca** [comment]
@mhdawson I'd also update the banner
https://github.com/nodejs/nodejs.org/blob/178a3c641465e432bd4c96d301094a1413520583/build.js#L284

**Comment by MylesBorins** [comment]
@mhdawson  afaik we've updated the banner on all previous sec announcements. @lpinca graciously pointed out where to do this above

**Review by cjihrig — APPROVED** [review]

File: locale/en/blog/vulnerability/december-2017-security-releases.md (hunk 2 above)
Comment (mhdawson) [reply locale/en/blog/vulnerability/december-2017-security-releases.md]: Fixed

**Comment by mhdawson** [comment]
Also added to banner.

**Review by evanlucas — APPROVED** [review]

**Review by pierreneter — APPROVED** [review]

**Review by jseijas — APPROVED** [review]

**Comment by mhdawson** [comment]
Landed as e552ce42ede5fe45d8cbaefaa9eb24359e0998a8

**Comment by Trott** [comment]
Oops, I was a few minutes too late. Oh, well, they were very small nits.

**Pull request closed without merging**